example_project/users/views.py

```diff
@@ -8,6 +8,7 @@
 from rest_framework.response import Response
 from rest_social_auth.serializers import UserSerializer
 from rest_framework.authentication import SessionAuthentication, TokenAuthentication
+from rest_framework_jwt.authentication import JSONWebTokenAuthentication
```

Comment:
But this view is in example_project...
I think we just need to add dependency for tests, i'll do it soon.